commec/screeners/check_reg_path.py

```diff
@@ -15,7 +15,7 @@
 import sys
 import textwrap
 import pandas as pd
-from commec.tools.blast_tools import readblast, trimblast, taxdist, tophits
+from commec.tools.blast_tools import readblast, get_taxonomic_labels, get_top_hits
```

Comment:
thank you for renaming tophits and taxdist, perhaps we should quickly change readblast to read_blast snake casing whilst we're here?